server/service/src/invoice_line/stock_out_line/mod.rs

```diff
@@ -35,9 +35,9 @@ impl StockOutType {
 }
 
 pub(crate) fn invoice_backdated_date(invoice: &InvoiceRow) -> Option<NaiveDateTime> {
-    if let Some(allocated_date) = invoice.allocated_datetime.clone() {
-        if allocated_date < invoice.created_datetime {
-            return Some(allocated_date);
+    if let Some(backdated_datetime) = invoice.backdated_datetime.clone() {
+        if backdated_datetime < invoice.created_datetime {
```

Comment:
I think this logic is still good, if backdated time is after created_datetime, it's sort of considered not a backdated invoice, although the field is populated in the db which could cause some confusion in reporting (if that was to happen) e.g. # of backdated invoiced could be incorrect if people set it to today a lot...

Reply:
> There is an edge case that if someone does manually change their prescription to as today, it will be treated as backdated,

With this change that other edge case is dealt with ?

Reply:
@andreievg  I think we agreed to leave this for now? I'll raise an issue for follow up though...